reference/pipeline/stages/index.md

```diff
@@ -89,15 +89,23 @@ Server Groups and any existing Server Groups. Enabling a Server Group also
 re-enables auto-scaling policies, if applicable.
 
 ### Find Artifact From Execution
-Find and bind an artifact from another pipeline execution.
+Find and bind an artifact from a different execution to the current one,
+including artifacts from different Spinnaker applications. To do this, specify
+the type of artifact (GCS, GitLab, Docker, etc) and its path. You can specify
+that Spinnaker should only consider executions that meet certain criteria, such
+as ones that have completed successfully or that are currently running.
+Spinnaker will always return the artifact from the most recent execution that
+matches your criteria.
 
 ### Find Image From Cluster
 Find an image to deploy from an existing Cluster. Make sure that you specify the
 Cluster and Server Group such that there is exactly one match, or this may
 behave in unexpected ways.
 
 ### Find Image From Tags
-Find an image to deploy from tags.
+Find the newest image that matches all specified tags. Tags can only contain
```

Comment:
[This](https://www.spinnaker.io/guides/user/tagging/) kind of tag? Or a tag from the repo? Or Docker registry?

Reply:
Ah. Tag as described below under Tag Image, right? If so, maybe link to that?

Reply:
Expanded the tagging description to try to clarify briefly -- let me know what you think. 

We may also want to add some documentation somewhere about how Spinnaker interacts with image tags, since I don't think that exists anywhere.